plugin_store/api/__init__.py

```diff
@@ -76,6 +77,12 @@ async def plugins_list(
     plugins = await db.search(db.session, query, tags, hidden)
     return plugins
 
+@app.post("/increment")
+async def submit_release(
```

Comment:
Duplicated view name. You  probably meant `download_increment` or something.

Reply:
oops, interesting that it still worked though.